node/checkURL.js

```diff
@@ -0,0 +1,9 @@
+function checkURL(url) {
```

Comment:
since we went with `purifyHtml` instead of `purifyHTML` I think we should to with `checkUrl`

```suggestion
function checkUrl(url) {
```

Reply:
okay